.golangci.yml

```diff
@@ -82,6 +82,10 @@ issues:
         - forbidigo
       # this is the one file we want to allow exec.Cmd functions in
       path: private/pkg/command/runner.go
+    - linters:
+        - forbidigo
+        # we use os.Rename here to rename files in the same directory
+      path: private/bufpkg/bufmodule/bufmodulecache/cas_module_cacher.go
```

Comment:
We banned rename for a reason, can we provide justification as to why this case does not hit that reason (add in comments)

Reply:
Yep - see https://github.com/bufbuild/buf/pull/1847#discussion_r1110431719.

Given a temp dir, i.e. `~/.cache/buf/v1/module/{remote}/{owner}/{repo}/blobs`, we are creating a temporary file in `~/.cache/buf/v1/module/{remote}/{owner}/{repo}/blobs/{digest[:2]}/` and then renaming it to `~/.cache/buf/v1/module/{remote}/{owner}/{repo}/blobs/{digest[:2]}/digest[2:]`. This won't cross filesystem boundaries since we are creating the temp file in the same directory where it will ultimately live.